bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/form/FormGroup.java

```diff
@@ -31,31 +30,11 @@
  */
 public class FormGroup extends Border {
 
-    /**
-     * Holder class for all possible form group sizes
-     */
-    public enum Size implements ICssClassNameProvider {
```

Comment:
Let's just deprecate the enum. It will be removed later, probably in a major version.